apps/avifdec.c

```diff
@@ -45,6 +45,9 @@ static void syntax(void)
     printf("    --ignore-icc      : If the input file contains an embedded ICC profile, ignore it (no-op if absent)\n");
     printf("    --size-limit C    : Specifies the image size limit (in total pixels) that should be tolerated.\n");
     printf("                        Default: %u, set to a smaller value to further restrict.\n", AVIF_DEFAULT_IMAGE_SIZE_LIMIT);
+    printf("  --dimension-limit C : Specifies the image dimension limit (width or height) that should be tolerated.\n");
+    printf("                        Default: %u, set to a smaller value to further restrict, or 0 to ignore.\n",
```

Comment:
I just noticed a copy-and-paste error here. "set to a smaller value to further restrict" is correct for `--size-limit` because we don't allow `--size-limit` to be set to a value larger than the default value. But there is no such constraint for `--dimension-limit`. I will change this line to:

```
    printf("                        Default: %u, set to 0 to ignore.\n",
```